app/routers/organizations.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, status, HTTPException
+from database import client
+from models import Organization
+
+router = APIRouter()
+
+
+@router.get("/check-auth-token/{api_key}", response_model=Organization)
+async def get_auth_token(api_key: str):
+
+    if (
+        org := client.quiz.organization.find_one({"org_key": api_key}, {"org_name": 1})
```

Comment:
why `{"org_name": 1}`?
Why is there a need to sort by org name?

Reply:
It's projection, not sorting. 
Instead of returning all the fields in the query, it returns only the mentioned field. 
In this case, rather than returning the id, name and api_key, it returns only the org name.

Reply:
understood
it was not super clear, maybe because of the syntax

I had two questions
- you're finding things here by `org_key` but I don't see any `org_key` defined in the model?
- why keep two separate names? `org_key` and `api_key` when they mean the same thing?